File: app/src/main/java/ca/com/toronto/utils/InternetConnection.java
package ca.com.toronto.utils;

import android.content.Context;
import android.net.ConnectivityManager;
import android.support.annotation.NonNull;

/**
 * Created by Royal_L on 22-Oct-17.
 */

public class InternetConnection {
    /**
     * CHECK WHETHER INTERNET CONNECTION IS AVAILABLE OR NOT
     */
    public static boolean checkConnection(@NonNull Context context) {
        return ((ConnectivityManager) context.getSystemService
                (Context.CONNECTIVITY_SERVICE)).getActiveNetworkInfo() != null;
    }
}

File: app/src/main/java/ca/com/toronto/utils/InternetConnection.kt
package ca.com.toronto.utils

import android.content.Context
import android.net.ConnectivityManager

/**
 * Created by Royal_L on 22-Oct-17.
 */
object InternetConnection {
    /**
     * CHECK WHETHER INTERNET CONNECTION IS AVAILABLE OR NOT
     */
    fun checkConnection(context: Context): Boolean {
        return (context.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager).activeNetworkInfo != null
    }
}
